spec/lib/simp/beaker_helpers_spec.rb

```diff
@@ -149,29 +146,29 @@ def initialize
       ENV['PUPPET_INSTALL_VERSION']= '5.5.0'
       @helper.host.options = {'puppet_collection' => 'puppet6'}
       expected = {
-      :puppet_install_version   => '5.5.0',
-      :beaker_puppet_collection => 'puppet5',
-      :puppet_install_type      => 'agent'
+        :puppet_install_version   => '5.5.0',
+        :puppet_collection        => 'puppet5',
+        :puppet_install_type      => 'agent'
       }
       expect( @helper.get_puppet_install_info ).to eq expected
     end
 
     it 'extracts info from BEAKER_PUPPET_AGENT_VERSION' do
       ENV['BEAKER_PUPPET_AGENT_VERSION']= '4.10.5'
       expected = {
-      :puppet_install_version   => '1.10.5',
-      :beaker_puppet_collection => nil,
-      :puppet_install_type      => 'agent'
+        :puppet_install_version   => '1.10.5',
+        :puppet_collection        => nil,
+        :puppet_install_type      => 'agent'
       }
       expect( @helper.get_puppet_install_info ).to eq expected
     end
 
     it 'extracts info from PUPPET_VERSION' do
       ENV['PUPPET_VERSION']= '4.10.5'
```

Comment:
Can't we just remove this test since p4 is past EoL? (And it is stripped from our tests in other modules as well)